index.js

```diff
@@ -408,13 +408,16 @@
     return BinaryType(name, functionUrl(name), test, _1, _2);
   }
 
+  //  applyParameterizedType :: (Type... -> Type) -> Type
+  function applyParameterizedType(type) {
+    return typeof type === 'function' ?
+      type.apply(null, Z.map(K(Unknown), range(0, type.length))) :
+      type;
+  }
+
```

Comment:
The reason was to enable the `test([], Type, applyParameterizedType(x))`-way of testing `VariadicType` membership.